jobrunner/create_or_update_jobs.py

```diff
@@ -279,33 +280,61 @@ def job_should_be_rerun(job_request, job):
 
 
 def assert_new_jobs_created(new_jobs, current_jobs):
-    if new_jobs:
-        return
-
-    pending = [job.action for job in current_jobs if job.state == State.PENDING]
-    running = [job.action for job in current_jobs if job.state == State.RUNNING]
-    succeeded = [job.action for job in current_jobs if job.state == State.SUCCEEDED]
-    failed = [job.action for job in current_jobs if job.state == State.FAILED]
+    tracer = trace.get_tracer("new_jobs")
+    with tracer.start_as_current_span("assert_new_jobs_created"):
+
+        span = trace.get_current_span()
+        if new_jobs:
+            span.set_attribute("new_jobs", True)
+            # want to have access to the job id that is causing these new jobs
+            # span.set_attribute("job_id", job.id)
+            span.set_attribute(
+                "new_job_actions", ", ".join(f"{job.action}" for job in new_jobs)
```

Comment:
```suggestion
                "actions", ", ".join(f"{job.action}" for job in new_jobs)
```
We already new these are new jobs, we can just call them actions?